tgui/packages/tgui/interfaces/RndConsole/SettingsMenu.js

```diff
@@ -0,0 +1,81 @@
+import { useBackend } from '../../backend';
+import { Box, Button, Flex, LabeledList, Section } from '../../components';
+
+export const SettingsMenu = (properties, context) => {
+  const { data, act } = useBackend(context);
+
+  const { sync, admin, linked_destroy, linked_lathe, linked_imprinter } = data;
+
+  return (
```

Comment:
I recommend pulling each of these section out into their own consts for readability sake